.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+COMMIT
```

Comment:
Yes but that wont happen during installation of released version, as there is no git directory